urllib3/util/retry.py

```diff
@@ -152,7 +156,8 @@ class Retry(object):
     def __init__(self, total=10, connect=None, read=None, redirect=None, status=None,
                  method_whitelist=DEFAULT_METHOD_WHITELIST, status_forcelist=None,
                  backoff_factor=0, raise_on_redirect=True, raise_on_status=True,
-                 history=None, respect_retry_after_header=True):
+                 history=None, respect_retry_after_header=True,
+                 remove_headers_on_redirect=None):
```

Comment:
Might be nicer if this were a constant value, like `remove_headers_on_redirect=DEFAULT_REDIRECT_HEADERS_BLACKLIST` or somesuch.

Reply:
Sure thing, that would be an easy addition.

Reply:
Off, I don't like `None` meaning `default` here. I assume you did this to avoid making the default parameter a mutable value - in which case, you can make the constant a `frozenset`.

Reply:
Is flake8 smart enough to know `frozenset` isn't a mutable value/doesn't report mutable default value for `frozenset`? I honestly don't know.

Reply:
Oh, duh, it's used above this parameter as a default so of course it works! I'll make this change.